models/portfolioCorrelation.js

```diff
@@ -0,0 +1,99 @@
+const Sequelize = require('sequelize')
+const config = require(__dirname + '/../config/db.json').mysql
+let sequelize = null
+sequelize = new Sequelize(config.database, null, null, config)
+
+sequelize.authenticate().then(() => {
+  console.log("connection established")
```

Comment:
use single quotes 'connection established'
do this every where
we follow this format for string to be single quoted